src/Form/Type/CustomUrlType.php

```diff
@@ -40,7 +66,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('url', TextType::class, [
                 'label' => $this->trans('URL', 'Modules.Linklist.Admin'),
                 'required' => true,
-                'constraints' => [new Assert\Url()],
+                'constraints' => [
+                    version_compare(Kernel::VERSION, '4.4') < 0
```

Comment:
```suggestion
                    isset($pattern)
```
The check as already been done above, checking for variable existence is faster

Reply:
Just checked the condition above, I don't get the $pattern variable, there is a const you can already change :thinking: